dashboard/app/helpers/pd/workshop_survey_results_helper.rb

```diff
@@ -489,7 +489,7 @@ def generate_facilitator_averages(summary)
       QUESTIONS_FOR_FACILITATOR_AVERAGES.each do |category, questions|
         facilitator_averages[facilitator][category.to_s.downcase.to_sym] = {}
         [:this_workshop, :all_my_workshops].each do |column|
-          average = (facilitator_averages[facilitator].slice(*(questions.map {|question| question[:primary_id]})).values.map {|x| x[column]}.reduce(:+) || 0) / questions.size.to_f
+          average = (facilitator_averages[facilitator].slice(*(questions.map {|question| question[:primary_id]})).values.map {|x| x[column]}.inject(0) {|sum, n| sum + (n || 0)} || 0) / questions.size.to_f
```

Comment:
Why the rename from `reduce` -> `inject`? My understanding is they are aliased to each other.

Would be great to clean up this (already very long line) into a more readable set of statements!